idm/idm_test.go

```diff
@@ -110,7 +110,7 @@ func TestUninitialized(t *testing.T) {
 }
 
 func TestAllocateInRange(t *testing.T) {
-	i, err := New(nil, "myset", 5, 10)
+	i, err := New(nil, "myset", 5, 12)
```

Comment:
yes since the allocation is serial and not the first available expanding the range to suit the test